docs/cloud/data-ingestion.md

```diff
@@ -21,3 +21,104 @@ If your Graphite stack does not currently contain any relay, then you can simply
 
 When creating a Hosted Metrics Graphite instance, we provide a carbon-relay-ng config file that you can plug in and be ready to use out of the box.
 We also have Grafana Labs engineers ready to advise further on set up, if needed.
+
+## Using carbon-relay-ng as a replacement for carbon-relay or carbon-cache
+
+The most simple way to send your carbon traffic to GrafanaCloud is to use carbon-relay-ng as a replacement for your current carbon-relay or carbon-cache. Carbon-relay-ng has a carbon input which supports the plain text and the pickle protocols, just like carbon-relay and carbon-cache. 
+
+An example configuration to do that can be downloaded from your Hosted Metrics instance's "Details" page. It contains a `grafanaNet` route pointing at your instance.
+
+## Sending a copy of your carbon traffic to GrafanaCloud
+
+It is possible to duplicate your data to send one copy to your existing Graphite infrastructure and the other to GrafanaCloud. To do this you can put an instance of carbon-relay-ng in front of your existing carbon-relay or carbon-cache and make it duplicate the traffic. Carbon-relay-ng allows you to specify routes of various types, to send a copy to GrafanaCloud you need to add a route of the type `grafanaNet`, to send a copy to your existing carbon-relay/carbon-cache you can add a carbon route.
+
+For example if you currently have carbon-relay listening on port `2003` and all of your infrastructure is sending its carbon traffic there, you could change it to listen on port `2053` instead and then start a carbon-relay-ng on port `2003` with this config to send a copy of the traffic to `localhost:2053`:
+
+```
+[[route]]
+key = 'carbon'
+type = 'sendAllMatch'
+destinations = [
+    'localhost:2053 spool=true pickle=false'
+]
+
+[[route]]
+key = 'grafanaNet'
+type = 'grafanaNet'
+addr = 'https://<Your endpoint address>'
+apikey = '<Your Grafana.com API Key>'
```

Comment:
Should we mention anything related to passing in secrets?
Are env vars an option?

Reply:
I don't think this currently supports specifying env vars, it will need to be the actual string of the key

Reply:
adding support for using env vars there might be a good idea